scripts/turtle_offspring_nesting.json

```diff
@@ -1,9 +1,17 @@
 {
-    "citation": "Lisa E. Schwanz, Rachel M. Bowden, Ricky-John Spencer, and Fredric J. Janzen. 2009. Nesting ecology and offspring recruitment in a long-lived turtle. Ecology 90:1709.",
-    "description": "",
+    "citation": "Lisa E. Schwanz, Rachel M. Bowden, Ricky-John Spencer, and Fredric J. Janzen. 2009. Nesting ecology and offspring recruitment in a long-lived turtle. Ecology 90:1709. [https://doi.org/10.6084/m9.figshare.3531323.v1]",
+    "description": "Valuable empirical resource for exploring important facets of nesting ecology and hatchling recruitment in a wild population of a long-lived species.",
     "homepage": "https://figshare.com/articles/Data_Paper_Data_Paper/3531323",
     "keywords": [
-        "Taxon > Animals"
+        "Taxon > Animals",
+        "demography",
+        "early-life-stages",
+        "fluctuating-environments",
```

Comment:
Have you tried to test the changes using the data retriever list by keyword command? 

And how about if we remove  `-` and just provide strings
`"demography", "early life stages", "fluctuating environments",......`

Reply:
@ss-is-master-chief 
    "keywords": [
        "Taxon > Animals",
        "demography",
        "early life stages",
        "fluctuating environments",
        "maternal effects",
        "nesting behavior",
        "predation",
        "temperature dependent sex determination",
        "turtles"

    ],